test/kitchen-sink/Row3.tsx

```diff
@@ -17,7 +21,13 @@ export default function Row3() {
         </ListBox>
       </ContextMenu>
       <CookieConsent position="bottom-right"></CookieConsent>
-      <Crud items={crudData}></Crud>
+      <Crud items={crudData}>
+        <FormLayout slot="form">
+          <TextField label="Name" {...crudFieldProps({ path: 'name' })} />
```

Comment:
This could also be simplified to: `<TextField label="Name" {...crudFieldPath('name')} />`

Reply:
Neat idea.

Reply:
I'd rather name it simply `path`: `<TextField label="Name" {...path('name')} />`.